package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "koa-body",
-  "version": "1.5.0",
+  "version": "1.6.0",
```

Comment:
@tunnckoCore I did it before and the maintainer was ok with it. If he complains I will remove it.